code/go/internal/validator/semantic/validate_minimum_kibana_version.go

```diff
@@ -52,11 +58,25 @@ func validateMinimumKibanaVersion(packageType string, packageVersion semver.Vers
 		return nil
 	}
 
-	if kibanaVersionConditionIsGreaterThanOrEqualTo8_8_0(kibanaVersionCondition) {
+	if kibanaVersionConditionIsGreaterThanOrEqualTo(kibanaVersionCondition, "8.8.0") {
 		return nil
 	}
 
-	return errors.New("Warning: conditions.kibana.version must be ^8.8.0 or greater for non experimental input packages (version > 1.0.0)")
+	return errors.New("conditions.kibana.version must be ^8.8.0 or greater for non experimental input packages (version > 1.0.0)")
```

Comment:
Removed `Warning: ` prefix since this error is always raised and fails the validation.